licences-specs/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
 
   testImplementation 'org.codehaus.groovy:groovy-all:2.5.14'
   testImplementation 'net.sourceforge.jtds:jtds:1.3.1'
-  testImplementation 'com.jayway.jsonpath:json-path:2.5.0'
+  testImplementation 'com.jayway.jsonpath:json-path:2.7.0'
```

Comment:
Had problems getting risk management version 2 integration tests to pass on circle so tried upgrading jsonpath to trigger invalidating the dependency cache which may have been causing the problem.